module/VuFindTheme/src/VuFindTheme/ResourceContainer.php

```diff
@@ -160,11 +184,46 @@ public function getCss()
     /**
      * Get Javascript files.
      *
+     * @param string $position Position where the files should be inserted
+     * (allowed values are 'header' or 'footer').
+     *
+     * @return array
+     */
+    public function getJs(string $position=null)
+    {
+        $this->js = array_unique($this->js, SORT_REGULAR);
```

Comment:
It works if SORT_REGULAR is used, according to the documentation (else all elements will be treated as strings per default).

Reply:
Thanks! I actually checked the documentation yesterday and didn't come away with the same interpretation on the first reading, but I think I see now how this could work. :-) The "Note: Two elements are considered equal if and only if (string) $elem1 === (string) $elem2 i.e. when the string representation is the same, the first element will be used. " confused me -- but I guess SORT_REGULAR makes that note irrelevant, and regular `==` comparison of non-string-cast arrays would be appropriate for our purposes.